railties/lib/rails/configuration.rb

```diff
@@ -64,9 +65,12 @@ def unshift(*args, &block)
       end
 
       def merge_into(other) #:nodoc:
-        @operations.each do |operation, args, block|
-          other.send(operation, *args, &block)
+        [@operations, @delete_operations].each do |operations|
```

Comment:
Instead of two loops can we just concat the `@opeartions` with `@delete_operations`?